features/config/developer_can_use_composer_to_create_suites.feature

```diff
@@ -9,6 +9,7 @@ Feature: Composer can be leveraged to create suites
       """
       composer_namespace_provider: true
       """
+      And the fake autoloader is setup
```

Comment:
I want a reader of just the feature to understand the relationship between When and Then

In the Gherkin it should be something like:
`And there is a PSR-4 namespace "MilkyWay\OrionCygnusArm\" configured for the "src" folder`

Currently the When/Then relationship is only understandable if you read the autoload file